Always disables X11 headers, so that other packages don't have to define MESA_EGL_NO_X11_HEADERS.

diff --git a/include/EGL/eglplatform.h b/include/EGL/eglplatform.h
index 4876dbd46..88fd370aa 100644
--- a/include/EGL/eglplatform.h
+++ b/include/EGL/eglplatform.h
@@ -118,24 +118,10 @@ typedef intptr_t EGLNativeWindowType;
 
 #elif defined(__unix__) || defined(__APPLE__)
 
-#if defined(MESA_EGL_NO_X11_HEADERS)
-
 typedef void            *EGLNativeDisplayType;
 typedef khronos_uintptr_t EGLNativePixmapType;
 typedef khronos_uintptr_t EGLNativeWindowType;
 
-#else
-
-/* X11 (tentative)  */
-#include <X11/Xlib.h>
-#include <X11/Xutil.h>
-
-typedef Display *EGLNativeDisplayType;
-typedef Pixmap   EGLNativePixmapType;
-typedef Window   EGLNativeWindowType;
-
-#endif /* MESA_EGL_NO_X11_HEADERS */
-
 #elif defined(__HAIKU__)
 
 #include <kernel/image.h>

--
2.22.1
